pep-0427.txt

```diff
@@ -158,10 +158,10 @@ package's basic interpreter requirements and are detailed in PEP 425.
 Escaping and Unicode
 ''''''''''''''''''''
 
-Each component of the filename is escaped by replacing runs of
-non-alphanumeric characters with an underscore ``_``::
-
-    re.sub("[^\w\d.]+", "_", distribution, re.UNICODE)
+The ``distribution`` and ``version`` components of the filename should
+follow the same rules as the ``.dist-info`` directory. [2]_ Each
```

Comment:
I think it's good practice for specifications to contain their own definitions with minimal references to other documents, even if that means some duplication. E.g. if something changed about dist-info directories, it may be unclear whether the change also affects wheel filenames. Admittedly this is somewhat pedantic!